core/ops_builtin_v8.rs

```diff
@@ -165,7 +166,12 @@ fn op_eval_context<'a>(
   let source = v8::Local::<v8::String>::try_from(source.v8_value)
     .map_err(|_| type_error("Invalid source"))?;
   let specifier = match specifier {
-    Some(s) => resolve_url_or_path_deprecated(&s)?.to_string(),
+    Some(s) => {
+      // TODO(bartlomieju): ideally we shouldn't need to call `current_dir()` on each
+      // call - maybe it should be caller's responsibility to pass fully resolved URL?
```

Comment:
Yes.

Reply:
Already covered in https://github.com/denoland/deno/pull/18173